packages/db/src/config.ts

```diff
@@ -33,24 +34,39 @@ export const getSecretConnectionConfig = async (
       throw new Error(`AWS Secret ${SecretId} is missing required key '${key}'`);
     }
   });
+  const rejectUnauthorized = dbAccessMeta.rejectUnauthorized !== 'false' && dbAccessMeta.rejectUnauthorized !== false;
+  const useSsl = dbAccessMeta.disableSSL === 'true' || dbAccessMeta.disableSSL === true;
```

Comment:
Same as above. This is probably fine but is less cool than being able to assume a type 😄

Reply:
See comment above - `useSsl` is assumed to be a bool by TS.